featurebyte/sql/tile_schedule_online_store.py

```diff
@@ -85,84 +86,85 @@ async def execute(self) -> None:
 
             logger.debug(f"{f_name}, {fs_table}, {f_entity_columns}, {f_value_type}")
 
-            entity_columns = (
-                [col.replace('"', "") for col in f_entity_columns.split(",")]
-                if f_entity_columns
-                else []
-            )
-
             # check if feature store table exists
             fs_table_exist_flag = await self.table_exists(fs_table)
             logger.debug(f"fs_table_exist_flag: {fs_table_exist_flag}")
 
-            entities_fname_str = ", ".join(
-                [self.quote_column(col) for col in entity_columns + [f_name]]
+            quoted_result_name_column = self.quote_column(
+                InternalName.ONLINE_STORE_RESULT_NAME_COLUMN
+            )
+            quoted_value_column = self.quote_column(InternalName.ONLINE_STORE_VALUE_COLUMN)
+            quoted_entity_columns = (
+                [self.quote_column(col.replace('"', "")) for col in f_entity_columns.split(",")]
+                if f_entity_columns
+                else []
+            )
+            column_names = ", ".join(
+                quoted_entity_columns + [quoted_result_name_column, quoted_value_column]
             )
 
             current_ts = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
 
             if not fs_table_exist_flag:
                 # feature store table does not exist, create table with the input feature sql
                 create_sql = construct_create_table_query(
-                    fs_table, f"select {entities_fname_str} from ({f_sql})", session=self._session
+                    fs_table,
+                    f"select {column_names} from ({f_sql})",
```

Comment:
Should we use sqlglot here?

Reply:
Ideally this should be using sqlglot. To minimise unrelated changes I think it's better to refactor that in separate PRs.